### DISCUSSION TITLE
@meteorrn/local offline sync

### DISCUSSION THREAD

**Discussion by ghost** [TheRealNate/react-native-meteor#42] [Q&A] [answered]
Hi there,

no frustration here, only the question how to get a grounddb replacement for RN.
Just asking on how the offline functionality is evolving? How can the local database be used then?
Regards

**Comment by not-ai-yet** [comment, marked as answer]
Hi @1seck, sorry about the delayed reply.  There is a GroundDB-like package for MeteorRN, called [`@meteorrn/local`](https://github.com/TheRealNate/meteor-react-native/tree/master/companion-packages/meteorrn-local). It has similar features, and using it you can run your app completely offline.
